policyengine_us/variables/gov/states/me/tax/income/credits/me_child_care_credit/me_child_care_credit.py

```diff
@@ -0,0 +1,48 @@
+from policyengine_us.model_api import *
+
+
+class me_child_care_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "ME child care credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_sched_a_ff.pdf"  # (y)
+    defined_for = StateCode.ME
+
+    def formula(tax_unit, period, parameters):
+        placeholder = (
+            0  # until I know how to define a new input for Step 4 Expenses
```

Comment:
If we do create an input for Step 4 expenses, I think we can create something similar to total childcare expenses (linked), but let's see what Max thinks.
https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/variables/household/expense/childcare/tax_unit_childcare_expenses.py

Reply:
It looks like the step can be 1, 2, 3, or 4. How about a `me_childcare_step` int variable that ranges from 1 (default) to 4, and has no formula, then check against that?

Reply:
I see that there is 4 possible steps, but I only see the filer having to specify for step 4 programs. We could have a int variable with the step, but we would still need to have there variables for the specific amounts of expenses per program. A filer may have used multiple child care programs with different qualifications. The form asks for 2 expense totals: Step 4 and non-Step 4 expenses. I started by creating a float variable for me_child_care_step_4_expenses.py and defaulting it at 0. Regular expenses are the federal child care expenses minus this step 4 amount in me_child_care_credit. Are you recommending asking for (1-4) step qualifications in anticipation where it may apply in other parts of the tax code or for future cases? 
<img width="920" alt="image" src="https://user-images.githubusercontent.com/29848542/226628438-6e78ae7d-4b6d-4ece-8bd9-0f398e7c58d0.png">